gcsio/src/test/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageTest.java

```diff
@@ -261,6 +273,55 @@ public void testCreateObjectWithGenerationId() throws Exception {
       assertThat(writtenData.toByteArray()).isEqualTo(testData);
     }
   }
+  /** Test the Statistics of http GET request */
+  @Test
+  public void testHttpGetRequestStat() throws IOException {
```

Comment:
please add assertions with read channels, Looks like we are just creating the writechannel and not accessing the read path